services/ocr/utils/recogniseTextOcrSpace.js

```diff
@@ -39,6 +39,7 @@ export const recogniseTextOcrSpace = async (fileName, language, logger) => {
         if (
             error.name === 'AxiosError'
             || error.message === 'socket hang up'
+            || error.message === 'Socket closed'
```

Comment:
Dispatching based on human-readable messages is unstable. Do they offer something like `error.code` with programmatically defined constants?